app/dockerdwrapper.c

```diff
@@ -484,6 +515,11 @@ start_dockerd(const struct settings *settings)
                               "-H unix:///var/run/docker.sock");
 
     g_strlcat(msg, " with IPC socket.", msg_len);
+  } else if (!use_ipc_socket && !use_tcp_socket) {
+    syslog(LOG_WARNING,
```

Comment:
Do this check early in the function - there is no need to set up a lot of other things that will not be used since we do not start dockerd. 
To be consistent with other parts of the code the log should be LOG_ERR
```suggestion
    syslog(LOG_ERR, "At least one of IPC socket or TCP socket must be set to \"yes\". dockerd will not be started.");
```